### PR TITLE
Implement simple gulp task to generate less files

### PR DESCRIPTION
**Pull request by mvantellingen**
Usage:
  npm install
  npm run compile
  npm run start

### CODE REVIEW

**Comment by mvantellingen** [comment]
@maikhoepfel perhaps this is a good starting point.

File: Makefile [hunk 1]
```diff
@@ -93,12 +93,8 @@ compiledmessages:
...
 
 css:
-	# Compile CSS files from LESS
-	lessc --source-map --source-map-less-inline src/oscar/static/oscar/less/styles.less src/oscar/static/oscar/css/styles.css
-	lessc --source-map --source-map-less-inline src/oscar/static/oscar/less/dashboard.less src/oscar/static/oscar/css/dashboard.css
-	# Compile CSS for demo site
-	lessc --source-map --source-map-less-inline sites/demo/static/demo/less/styles.less sites/demo/static/demo/css/styles.css
-	lessc --source-map --source-map-less-inline sites/demo/static/demo/less/responsive.less sites/demo/static/demo/css/responsive.css
+	npm install
+	npm run compile
```
Comment (maikhoepfel) [inline Makefile]: I know we used to call it `compile`, mostly out of ignorance. But I think a name like `preprocess-css` is much more appropriate.

**Comment by maikhoepfel** [comment]
Cool! And now live-reloading, linting of the JS, Babel for ES6... one can dream :+1: and this is a good foundation for it.

File: Makefile (hunk 1 above)
Comment (mvantellingen) [inline Makefile]: I'll change it

**Comment by mvantellingen** [comment]
Need to fix a couple of small things noted by a colleague and then i'll merge it

File: Makefile (hunk 1 above)
Comment (mvantellingen) [inline Makefile]: Ended up naming it 'build'. It's commonly used, and the task will possibly do non css related tasks in the future (javascript etc)

**Pull request merged**